libraries/app/api.cpp

```diff
@@ -594,11 +601,16 @@ namespace graphene { namespace app {
 
              if(node->operation_id(db).op.which() == operation_id)
                result.push_back( node->operation_id(db) );
-             }
+          }
           if( node->next == account_transaction_history_id_type() )
              node = nullptr;
           else node = &node->next(db);
        }
+       if( stop.instance.value == 0 && result.size() < limit ) {
```

Comment:
@ronakpatel3110  - What does this logic do ?

Reply:
@bobinson This change is part of https://github.com/bitshares/bitshares-core/pull/398. I had to add it because without this change testcases were failing. Also, Bitshares PR https://github.com/bitshares/bitshares-core/pull/398 is included in PeerPlays in #69.